datajoint/schemas.py

```diff
@@ -302,7 +376,8 @@ def list_tables(self):
 
 class VirtualModule(types.ModuleType):
     """
-    A virtual module which will contain context for schema.
+    A virtual module imitates a Python module representing a DataJoint schema from table definitions in the database.
+    It declares the schema objects and a class for each table.
```

Comment:
@dimitri-yatsenko We should update `VitrualModule` to respect the new `Schema().activate(create_schema=create_schema, create_table=create_table)` approach. Apparently for virtual modules now, `create_schema=False` is not being respected.